lib/geckoboard/dataset.rb

```diff
@@ -16,6 +16,10 @@ def put(data)
       client.put_data(id, payload_formatter.format(data))
     end
 
+    def post(data, options={})
```

Comment:
/cc @leocassarani thoughts on `append`? I think we've just been matching the HTTP verb for stuff up until now, but Its a good thought.